docs/guides/typescript.md

```diff
@@ -26,7 +26,7 @@ export const MyApp: React.SFC<{}> = () => {
       <h1>My Example</h1>
       <Formik
         initialValues={{ firstName: '' }}
-        onSubmit={(values: MyFormValues) => alert(JSON.stringify(values))}
+        onSubmit={(values: MyFormValues, actions: FormikActions<MyFormValues>) => console.log({ values, actions })}
```

Comment:
```suggestion
        onSubmit={(values: MyFormValues, actions: FormikActions<MyFormValues>) => {
            console.log({ values, actions });
            alert(JSON.stringify(values, null, 2));
            actions.setSubmitting(false)
         }}
```

Reply:
We should actually use the actions. Also, as of Formik 1.4.3 all of these types get inferred so you actually don't need these at all. 🤷‍♂️